privacy_bot/analysis/classifier.py

```diff
@@ -19,25 +34,39 @@
                     n_iter=50
                 ))
 ])
+url_clf = Pipeline([('tfidf', TfidfVectorizer(
+                    analyzer='char',
+                    ngram_range=(3, 5)
+                )),
+                ('clf', SGDClassifier())
+])
+
 
 if __name__ == "__main__":
+    args = docopt(__doc__)
+    print(args)
+    print(args['--entity'])
     labelled_dataset = Dataset.tars_to_labelled(
-        true_positives_path='privacy_policy_positive.tar.bz2',
-        true_negatives_path='privacy_policy_negative.tar.bz2'
+        true_positives_path=args['--tp'],
+        true_negatives_path=args['--tn'],
+        entity=args['--entity']
     )
-    
+    #TODO: training_size, testing_size into args (when > labelled data)
     training, test = labelled_dataset.training_testing(
```

Comment:
What about specifying a percentage instead of number? Does not seem very handy if you change the size of the dataset. We could just wrap this http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html#sklearn.model_selection.train_test_split from scikit directly and offer the same arguments.